nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,111 @@
+```bash
+npm install @microsoft/live-share@next --save
+```
+OR
+
+To install using yarn
+```bash
+yarn add @microsoft/live-share@next
+```
+
+#### Live Share media
+To install the latest version of the Live Share media package to your application:
+
+To install using npm
+
+```bash
+npm install @microsoft/live-share@next --save
+npm install @microsoft/live-share-media@next --save
+```
+OR
+
+To install using yarn
+
+```bash
+yarn add @microsoft/live-share@next
+yarn add @microsoft/live-share-media@next
+```
 
-```typescript
-import * as microsoftTeams from "@microsoft/teams-js";
+#### Live Share canvas
+
+To install the latest version of the Live Share media package to your application:
+
+To install using npm
+
+```bash
+npm install @microsoft/live-share@next --save
+npm install @microsoft/live-share-canvas@next --save
+```
+OR
+
+To install using yarn
+
+```bash
+yarn add @microsoft/live-share@next
+yarn add @microsoft/live-share-canvas@next
 ```
 
-You can also reference the entire library in html pages using a script tag.  There are three ways to do this:
+## Microsoft Teams JavaScript client library
+
+The Microsoft Teams JavaScript client library version `2.x.x` is enabled [run apps across Microsoft 365](/microsoftteams/platform/m365-apps/overview). The TeamsJS SDK is a superset of TeamsJS `1.x` versions: and supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Microsoft 365. Refer to the [Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library) overview for guidance on building Microsoft 365-enabled apps.
+
+### Finding the library
 
-> [!IMPORTANT]
-> Do not copy/paste these `<script src=...` URLs from this page; they refer to a specific version of the SDK. To get the `<script src=...></script>` markup for the latest version, always go to https://www.npmjs.com/package/@microsoft/teams-js.
+The TeamsJS client is distributed as an [npm package](www.npmjs.com/package/@microsoft/teams-js) and can be installed using npm or yarn.
 
-```html
-<!-- Microsoft Teams JavaScript API (via CDN) -->
-<script
-  src="https://res.cdn.office.net/teams-js/2.0.0/js/MicrosoftTeams.min.js"
-  integrity="sha384-Q2Z9S56exI6Oz/ThvYaV0SUn8j4HwS8BveGPmuwLXe4CvCUEGlL80qSzHMnvGqee"
-  crossorigin="anonymous"
-></script>
+To install using npm
 
-<!-- Microsoft Teams JavaScript API (via npm) -->
-<script src="node_modules/@microsoft/teams-js@2.0.0/dist/MicrosoftTeams.min.js"></script>
+  ```bash
+npm install --save @microsoft/teams-js
+  ```
+OR
 
-<!-- Microsoft Teams JavaScript API (via local) -->
-<script src="MicrosoftTeams.min.js"></script>
+To install using yarn
+
+```bash
+yarn add @microsoft/teams-js`
 ```
+## Microsoft Teams Fx library
```

Comment:
You might consider having an overarching section about MS Teams that introduces both these libraries, then have subsections that discuss them (and their installation) individually.